README.md

```diff
@@ -1,7 +1,6 @@
 <img src="logoConcord.png"/>
 
-
-# Concord-BFT: a Distributed Trust Infrastructure
+# Concord-BFT: A Distributed Trust Infrastructure
```

Comment:
nit - the word "infrastrcuture" is not countable, so I'd suggest to rearrange the title to sound a bit better in English - such as "Infrastructure for Distributed Trust Systems". WDYT?